lib/index.js

```diff
@@ -1,2 +1,58 @@
-// This file is in the entry point in your webpack config.
 import './styles.scss'
+const artistInput = document.getElementById("search-input")
+const findSongsBtn = document.getElementById("find-song-btn")
+const apiKey = "869c7b19121345facdc061b2aa12dabf"
+var $ = require("jquery");
```

Comment:
if we have time - I would recommend looking into ENV variables so api_key is not just hanging out

Reply:
I will get on this right away.